### ISSUE TITLE
SpyGlass: False detection, and Arma 3 crash

### ISSUE THREAD

**Issue by Jawshy** [ArmaLife/Altis#247]
Player was kicked by SpyGlass for accessing the physical 'I' inventory. Trying to reproduce the SpyGlass kick caused _Arma 3_ to crash due to being "Out of memory." 

**Snippets of logs**: 
###### remoteExec
```
17.02.2016 15:14:30: Jawsh (10.0.0.7:2316) 12345678901234567890123456789012 - #0 "spy_fnc_cookiejar ["Jawsh","12345678901234567","MenuBasedHack_RscDisplayInventory_Controls_87"]"
17.02.2016 15:14:30: Jawsh (10.0.0.7:2316) 12345678901234567890123456789012 - #0 "spy_fnc_notifyadmins ["Jawsh","Menu Hack: RscDisplayInventory number of controls do not match (Count 87)"]"
```

###### Client/local.rpt
```
15:14:05 "------------------------------------------------------------------------------------------------------"
15:14:05 "                End of Altis Life Client Init :: Total Execution Time 1.82495 seconds "
15:14:05 "------------------------------------------------------------------------------------------------------"
15:14:30 Overflow
15:14:30 Overflow
15:14:30 Overflow
15:14:34 Virtual memory total 4095 MB (4294836224 B)
15:14:34 Virtual memory free 165 MB (173633536 B)
15:14:34 Physical memory free 5072 MB (5319110656 B)
15:14:34 Page file free 31611 MB (33147265024 B)
15:14:34 Process working set 3320 MB (3481755648 B)
15:14:34 Process page file used 3569 MB (3742392320 B)
15:14:34 Longest free VM region: 3932160 B
15:14:34 VM busy 4138754048 B (reserved 169189376 B, committed 3969564672 B, mapped 141619200 B), free 156082176 B
15:14:34 Small mapped regions: 150, size 626688 B
15:14:37 Link to bc05250e (Obj-164,480:1294) not released
15:14:37 Link to bc05150d (Obj-162,480:1293) not released
15:14:37 Link to b6e3bbb9 (Obj-119,439:953) not released
15:14:37 Link to b723b3ba (Obj-118,441:954) not released
```

[Link to zip file](http://puu.sh/naWiL/23eba41bc3.zip) containing: 
- server.rpt
- client/local.rpt
- BattlEye logs
- extDB logs
- Error message screenshots

Game version: 1.54.133741
Mission version: v4.4r1 - latest commit: 32ef7e5fe41aec8cffb73a441d73c21e4f2a4e9b

**Comment by ColinM9991** [comment]
The crash is because of the memory leak Tonic forced or introduced with SpyGlass, if it's still implemented.

**Comment by tkcjesse** [comment]
Use good BEFilters, not spyglass imo...

**Comment by NiiRoZz** [comment]
Fixed cf909c26ba2e194987bb8749537a1a0f364d1cb0
